amida-tech/indaba-client
# Task/inba 332 logout and multi subjects

#### What's this PR do?
Creates a new logout link that removes the auth cookies and resets the states. 

Also returns functionality for adding multiple UOAs in the 

#### Related JIRA tickets:

#### How should this be manually tested?
Run the similarly named Greyscale branch.
Login with any user. Click "logout"
Ensure that user is returned to the login screen, and that the reducers and cookies are cleaned.

Log back in.
Create a new project.
Go to "Add Subjects"
Enter multiple subjects, divided by commas. Feel free to use single quotes and both add spaces or not between subjects.
Submit subjects.
Ensure that the list shows the newly entered subjects.
Check the tables 'ProductUOA' and 'UnitsOfAnalysis' for new entries.

#### Any background context you want to provide?
Caution: This PR probably breaks the add subject for existing projects. I am aware and will fix shortly. 

#### Screenshots (if appropriate):
